docs/en_US/Tutorial/NniOnWindows.md

```diff
@@ -1,6 +1,6 @@
 # NNI on Windows (experimental feature)
 
-Currently we support local, remote and pai mode on Windows. Windows 10.1809 is well tested and recommended.
+Running NNI on Windows is an experimental feature. Windows 10.1809 is well tested and recommended.
```

Comment:
Is NNI on Windows still experimental?

Reply:
@SparkSnail I think our windows version only contains part of the function of nni?

Reply:
> @SparkSnail I think our windows version only contains part of the function of nni?

Yes, NNI only support `local/remote/pai` on windows, I don't know if we will support all of platform in the feature. In this point, maybe it's reasonable that we call it an experimental feature.

Reply:
could remove this sentence. could be replaced with `We have supported NNI on Windows`.

Reply:
But we did not support all functions in Windows.  We should point out which functions we support, which not.